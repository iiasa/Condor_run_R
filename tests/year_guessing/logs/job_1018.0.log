000 (1018.000.000) 12/31 23:59:56 Job submitted from host: <147.125.52.253:9618?addrs=147.125.52.253-9618&noUDP&sock=8936_e446_3>
...
001 (1018.000.000) 12/31 23:59:58 Job executing on host: <147.125.99.213:9618?addrs=147.125.99.213-9618&noUDP&sock=12020_32f4_4>
...
006 (1018.000.000) 01/01 00:01:58 Image size of job updated: 19968
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
005 (1018.000.000) 01/01 00:03:58 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	284419  -  Run Bytes Sent By Job
	948  -  Run Bytes Received By Job
	284419  -  Total Bytes Sent By Job
	948  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        1         1 
	   Disk (KB)            :      307      402    468688 
	   Memory (MB)          :        0     1000      1024 
...
