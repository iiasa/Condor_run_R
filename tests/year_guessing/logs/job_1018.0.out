Machine = "limpopo3.iiasa.ac.at"
_CONDOR_SLOT = slot2_5
E:\condor\execute\dir_12436
